components/StyledVisuallyHidden.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+
+const VisuallyHidden = styled.span({
+    position: 'absolute !important',
+    width: '1px !important',
+    height: '1px !important',
+    margin: '0 !important',
+    padding: '0 !important',
+    border: '0 !important',
+    overflow: 'hidden !important',
+    clip: 'rect(0 0 0 0) !important',
+})
+
+export { VisuallyHidden };
```

Comment:
Apparently it's wrong but.. I learned that `!important` should be avoided 🤔

Reply:
No, it definitely should be avoided – so you can use it only when it's _really_ important 😉 In this case, it is crucial to "win" against other CSS so the text is indeed hidden (but present in the HTML for machines, e.g. assistive technology and search engines).